src/box/schema.h

```diff
@@ -100,6 +101,12 @@ schema_find_id(uint32_t system_space_id, uint32_t index_id, const char *name,
 #if defined(__cplusplus)
 } /* extern "C" */
 
+/**
+ * Returns true if feature is prohibited on the current schema version.
+ */
+bool
+feature_is_blocked(enum blocked_feature feature);
+
```

Comment:
Bad name:
- Unclear that it sets diag.
- Unclear what kind of feature it's about.

I'd name it `schema_check_feature` and make it return 0 or -1.

Reply:
Renamed